src/OpenTelemetry/Trace/TracerProviderBuilderSdk.cs

```diff
@@ -132,9 +132,9 @@ internal TracerProviderBuilder AddProcessor(BaseProcessor<Activity> processor)
         /// <summary>
```

Comment:
I think the code comments should use a plural tense. Here's an alternative:
```csharp
        /// <summary>
        /// Adds a listener for <see cref="Activity"/> objects created with the given operation name to the <see cref="TracerProviderBuilder"/>.
        /// </summary>
        /// <remarks>
        /// This is provided to capture legacy <see cref="Activity"/> objects creared without using the <see cref="ActivitySource"/> API.
        /// </remarks>
        /// <param name="tracerProviderBuilder"><see cref="TracerProviderBuilder"/> instance.</param>
        /// <param name="operationName">Operation name of the <see cref="Activity"/> objects to capture.</param>
        /// <returns>Returns <see cref="TracerProviderBuilder"/> for chaining.</returns>
        public static TracerProviderBuilder AddLegacyActivity(this TracerProviderBuilder tracerProviderBuilder, string operationName)
```

Reply:
I have updated the code comments.